troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -21,29 +26,42 @@ export default React.createClass({
         showAvailability: React.PropTypes.bool
     },
 
-    getDefaultProps: function() {
+    getDefaultProps() {
         return {
             showAvailability: true,
             editable: true
         }
     },
 
-    onEditClicked: function() {
+    onCardClick() {
+        let isOpen = this.state.isOpen
+            ? false : true;
```

Comment:
I think the reason that Connor is mentioning `!!` and `Boolean(...)` is because what you have coded up here looks like you're trying to get `isOpen` to be a boolean value. 

You could _toggle_ while using the approach that Connor is mentioning:
```
let isOpen = !Boolean(this.state.isOpen);
```